### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Name, Role and Value] [MAS4.1.2][HighImpact][Incorrect state is announced when focus moves onto Editor Field as Editing.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#707]
**User Experience:**
User who rely on AT will not get proper information about the elements, if  state is not announced properly which provides the state of control.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Narrator
**Attachments:**
[MAS4.1.2_Incorrect state.zip](https://github.com/OfficeDev/script-lab/files/1453895/MAS4.1.2_Incorrect.state.zip)

**Repro Steps:**
1.	Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2.	Navigate to “report generation” Iframe  and Turn on Narrator.
3.	Verify the state(editing) announced by screen reader when focus moves into editor field .

**Current Behavior:**
Incorrect state is announced when focus moves onto Editor Field. screen reader is announcing  as editing but the code is not editable, it will confuse user and may lead user to type the text .

**Expected Behavior:**
State should be Read only for the code that is present in the editor of Iframe. 

**Note**
Issue is repro for the Entry Point Page-Editor Field.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
